### PR TITLE
Added some css changes to fix navigation for search quick access

### PR DESCRIPTION
**Pull request by Rubemlrm**
Possible fix for suggestion #475 , now navbar is fixed for search quick access and quick access to other stuff on nav.

### CODE REVIEW

**Comment by phanan** [comment]
Bad guy here. If we can reduce the height of the bar, that would be great. It's taking too much space if you ask me.

File: themes/vue/source/js/common.js [hunk 1]
```diff
@@ -124,28 +124,19 @@
...
+        offset: 0
       })
     }
 
     // listen for scroll event to do positioning & highlights
     window.addEventListener('scroll', updateSidebar)
     window.addEventListener('resize', updateSidebar)
 
-    function updateSidebar () {
+    function updateSidebar (event) {
```
Comment (posva) [inline themes/vue/source/js/common.js]: We can remove the argument

**Comment by chrisvfritz** [comment]
@phanan We have the technology. 😄  What were you thinking? It's at 25px vertical padding right now. I agree it may be a bit large for a fixed header right now, but I also don't want to make it too cramped.

File: themes/vue/source/js/common.js (hunk 1 above)
Comment (chrisvfritz) [inline themes/vue/source/js/common.js]: Just removed this.

**Comment by chrisvfritz** [comment]
@phanan I just updated it to 20px. You can let me know what you think. It's also now possible to just change [this variable](https://github.com/vuejs/vuejs.org/pull/487/files#diff-43c2044b37446102035d286329cd5b8bR22) to play around with that value.

**Comment by phanan** [comment]
@chrisvfritz IMHO 10px looks just nice:

image

But I guess what I really mean is, the new padding should only apply when the menu detaches into the `fixed` position, maybe with a smooth transition.

**Comment by chrisvfritz** [comment]
Yeah... I was afraid you'd say that, but I agree. I'll try to update tomorrow.

**Pull request merged**